src/App/Resources/AppResources.de.resx

```diff
@@ -363,7 +363,7 @@
     <value>Fingerabdruck überprüfen</value>
   </data>
   <data name="VerifyMasterPassword" xml:space="preserve">
-    <value>Masterpasswort überprüfen</value>
```

Comment:
Bitwarden uses Crowdin for localisation, it would be better to submit this change there: https://github.com/bitwarden/mobile/blob/master/CONTRIBUTING.md